### PR TITLE
attached logic to buttons

### PR DESCRIPTION
**Pull request by Alyona1305**
Attached logic to delet button

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,25 +1,127 @@
...
+    `https://randomuser.me/api/?results=${quantity}`
+  );
+  const json = await response.json();
+  return json;
+}
+const elem = document.querySelector(".elem");
+
+document.querySelector(".buttonGet").onclick = function () {
+  const url = document.querySelector(".textButton").value;
+  console.log(url);
```
Comment (BooRex) [inline index.js:12]: remove all console logs

File: index.html [hunk 2]
```diff
@@ -8,32 +8,7 @@
...
-    <td> </td>
-</tr>
-<tr>
-    <td class='img'> <img src=''> </td>
-    <td class='fullName'> </td> 
-    <td class='age'> </td>
-    <td> </td>
-</tr>
-    </table>
+    <div class='elem'></div>
```
Comment (BooRex) [inline index.html:11]: why did you name it elem?
If you going to put there the table, please name it like table-container

File: index.js [hunk 3]
```diff
@@ -1,25 +1,127 @@
...
+  const response = await fetch(
+    `https://randomuser.me/api/?results=${quantity}`
+  );
+  const json = await response.json();
+  return json;
+}
+const elem = document.querySelector(".elem");
+
+document.querySelector(".buttonGet").onclick = function () {
+  const url = document.querySelector(".textButton").value;
```
Comment (BooRex) [inline index.js:11]: why this variable named url?
Please double check what you're getting from textInput?
Please change to correct name

File: index.html [hunk 4]
```diff
@@ -8,32 +8,7 @@
...
     <input type='text' value='2' class='textButton'>
```
Comment (BooRex) [inline index.html:12]: why text input named as textButton? Please change to correct

File: index.js [hunk 5]
```diff
@@ -1,25 +1,127 @@
...
+  console.log(url);
+  getResults(url).then((data) => {
+    console.log(data);
+    let arr = data.results;
+    createTable(elem, arr);
```
Comment (BooRex) [inline index.js:16]: you don't need arr variable, just put data inside function like
createTable(elem, data.results);

File: index.js [hunk 6]
```diff
@@ -1,25 +1,127 @@
...
+        break;
+      case 2:
+        th.innerHTML = "age";
+        break;
+      case 3:
+        th.innerHTML = "from";
+        break;
+    }
+
+    // if (i == 0) {
```
Comment (BooRex) [inline index.js:49]: please remove comments

File: index.js [hunk 7]
```diff
@@ -1,25 +1,127 @@
...
+        th.innerHTML = "photo";
+        break;
+      case 1:
+        th.innerHTML = "name";
+        break;
+      case 2:
+        th.innerHTML = "age";
+        break;
+      case 3:
+        th.innerHTML = "from";
```
Comment (BooRex) [inline index.js:45]: Those string shoud start from Upper case, like From Age Name

File: index.js [hunk 8]
```diff
@@ -1,25 +1,127 @@
...
+    thr.appendChild(th);
+  }
+
+  thead.appendChild(thr);
+  table.appendChild(thead);
+
+  const tbody = document.createElement("tbody");
+
+  for (let user of arr) {
+    console.log(user);
```
Comment (BooRex) [inline index.js:67]: remove console.log

File: index.js [hunk 9]
```diff
@@ -1,25 +1,127 @@
...
+          break;
+        case 2:
+          td.innerHTML = user.dob.age;
+          break;
+        case 3:
+          td.innerHTML = `${user.location.country}, ${user.location.city}`;
+          break;
+      }
+
+      // if (i == 0) {
```
Comment (BooRex) [inline index.js:90]: remove comments

File: index.js [hunk 10]
```diff
@@ -1,25 +1,127 @@
...
-    console.log(url)
-    getResults()
-    .then((data) => {
-        console.log(data);
-        let arr = data.results;
-        console.log(arr)
-    })
-}
\ No newline at end of file
+//      const table = document.createElement('table');
```
Comment (BooRex) [inline index.js:112]: remove those comments

File: index.js [hunk 11]
```diff
@@ -1,25 +1,127 @@
...
+    createTable(elem, arr);
+  });
+};
+
+document.querySelector(".buttonDelete").onclick = function () {
+  elem.innerHTML = "";
+};
+
+function createTable(parent, arr) {
+  const cols = 4;
```
Comment (BooRex) [inline index.js:25]: please don't rely on number 4
What if columns number will be 10?
So you need to add parameter for function

File: index.js (hunk 9 above)
Comment (Alyona1305) [reply index.js:90]: i want leave comments for youtself

File: index.css [hunk 12]
```diff
@@ -50,3 +50,16 @@ td {
   font-family: "Montserrat", sans-serif;
   font-weight: 800;
 }
+
```
Comment (BooRex) [inline index.css:53]: Currenlty you have 2 different styles for table tr and td.
Please combine them

File: index.js [hunk 13]
```diff
@@ -1,25 +1,127 @@
...
-const json = await response.json()
-return json
+      switch (i) {
+        case 0:
+          const img = document.createElement("img");
+          img.src = user.picture.large;
+          td.appendChild(img);
+          break;
+        case 1:
+          td.innerHTML = `${user.name.title} ${user.name.first} ${user.name.last}`;
```
Comment (BooRex) [inline index.js:80]: could you please read about destructurization?
https://learn.javascript.ru/destructuring
You can upgrade your code like
`const {first, last, title} = user.name;
td.innerHTML = `${title} ${first} ${last}`;
`

File: index.js [hunk 14]
```diff
@@ -1,25 +1,127 @@
...
+          break;
+        case 2:
+          td.innerHTML = user.dob.age;
+          break;
+        case 3:
+          td.innerHTML = `${user.location.country}, ${user.location.city}`;
```
Comment (BooRex) [inline index.js:86]: the same here with `user.location`

**Review by BooRex — APPROVED** [review]

**Pull request merged**